nipype/interfaces/tests/test_bids.py

```diff
@@ -0,0 +1,50 @@
+import os
+import json
+import sys
+
+import pytest
+from nipype.interfaces.bids_utils import BIDSDataGrabber
+from nipype.utils.filemanip import dist_is_editable
+
+have_pybids = True
+try:
+    import bids
+    from bids import grabbids as gb
+    filepath = os.path.realpath(os.path.dirname(bids.__file__))
+    datadir = os.path.realpath(os.path.join(filepath, 'grabbids/tests/data/'))
+except ImportError:
+    have_pybids = False
+
+
+# There are three reasons these tests will be skipped:
+@pytest.mark.skipif(not have_pybids,
+                    reason="Pybids is not installed")
+@pytest.mark.skipif(sys.version_info < (3, 0),
+                    reason="Pybids no longer supports Python 2")
+@pytest.mark.skipif(not dist_is_editable('pybids'),
```

Comment:
why is this bit necessary?

Reply:
from the initial #2174 - that's how the testing data was done

also #2207